test/e2e/page-objects/pages/header-navbar.ts

```diff
@@ -92,6 +85,59 @@ class HeaderNavbar {
       text: expectedLabel,
     });
   }
+
+  /**
+   * Verifies that the displayed account balance in modal is correct.
+   *
+   * @param params - An object containing the balance and currency.
+   * @param params.balance - The expected balance of the account.
+   * @param params.currency - The expected currency of the account balance.
+   */
+  async check_accountBalance({
```

Comment:
can we use the method in accountList instead : https://github.com/MetaMask/metamask-extension/blob/edd7b25ba7b912768341b6f34c369b1ca409b1c4/test/e2e/page-objects/pages/account-list-page.ts#L473
since we open accountlist page when we click the account in header

Reply:
woah nice catch! makes sense, fixing this now :pray: thank you!!